app/lib/frontend/handlers.dart

```diff
@@ -465,6 +547,13 @@ Future<shelf.Response> packageVersionHandlerHtml(
   return htmlResponse(cachedPage);
 }
 
+/// Handles requests for /experimental/packages/<package>/versions/<version>
+Future<shelf.Response> experimentalPackageVersionHandlerHtml(
+    shelf.Request request, String packageName, String versionName) {
+  return _packageVersionHandlerHtml(request, packageName, versionName,
+      templateService.renderNewPkgShowPage, false);
```

Comment:
I would prefer to have a `backed.uiPackageCacheV2` which can be used for caching in the new v2 handlers and invalidation is invalidating `backed.uiPackageCache` and `backend.uiPackageCacheV2`.

Then we don't need all these booleans.

Reply:
The flag in https://github.com/dart-lang/pub-dartlang-dart/pull/415 will substitute the bool here.